use-symfony-reproducer.md

```diff
@@ -0,0 +1,166 @@
+# Using a Symfony Reproducer
+
+***TIP
+This is part 2 of a 2-part _Reproducer_ series. If you haven't already, check
+out [Part 1](https://symfonycasts.com/blog/symfony-reproducer) first.
+***
+
+Ok, so you've created a reproducer app for a bug you've found in Symfony. For
+this article, we'll assume the reproducer you've created in part 1 is located
+at `~/my-reproducer`.
+
+## Forking the Symfony Repository
+
+The first thing we'll do is _fork_ the [`symfony/symfony`](https://github.com/symfony/symfony)
+repository on GitHub. This will give us our own copy of the Symfony repository
+where we can work.
+
+1. Visit https://github.com/symfony/symfony and click the _Fork_
+   button at the top of the page.
+2. Choose your personal GitHub account as the owner and keep the default name
+   of `symfony`.
+3. Leave all the other options as is and click _Create Fork_. You should now be on https://github.com/your-username/symfony.
```

Comment:
I see you also say to wait in 4th step, maybe we should split this step into 2 steps, i.e. write the part

> You should now be on https://github.com/your-username/symfony.

as a separate (5th) step?